core/model/modx/processors/system/clearcache.class.php

```diff
@@ -143,6 +143,7 @@ public function getPartitions() {
     public function clearByPaths() {
         $pathResults = array();
         /* deprecated: use a dedicated cache partition rather than specifying paths */
+        $this->modx->deprecated('2.1.4', 'Use a dedicated cache partition rather than specifying paths.', 'modSystemClearCacheProcessor clearByPaths');
```

Comment:
This method is called on each request, better to place this inside the `if (!empty($paths))` below so it only logs when the processor is called with paths to clear.